docs/api/covidcast-signals/quidel.md

```diff
@@ -119,13 +116,14 @@ p = 100 \left( \frac{N}{50} \frac{X}{N} + \frac{50 - N}{50}  \frac{X_s}{N_s} \ri
 $$
 where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
 taken with positive results taken in its parent state in the same time period.
+A parent state is defined as the state with the largest proportion of the population 
+in this county/MSA/HRR.
```

Comment:
@ryantibs confirming: if an msa or hrr crosses state boundaries, shrinkage only borrows samples from the state of greatest population within that msa/hrr, and not the others

Reply:
Resolved in teams leads. Sorry for the delay!